client/state/stats/lists/utils.js

```diff
@@ -14,3 +16,24 @@ import toPairs from 'lodash/toPairs';
 export function getSerializedStatsQuery( query = {} ) {
 	return JSON.stringify( sortBy( toPairs( query ), ( pair ) => pair[ 0 ] ) );
 }
+
+export const Parsers = {
+	/**
+	 * Returns a parsed payload from `/sites/{ site }/stats`
+	 *
+	 * @param  {Object} data    Stats query
+	 * @return {Object?}        Parsed stats data
+	 */
+	stats: ( data ) => {
+		if ( ! data || ! data.stats ) {
+			return null;
+		}
+
+		const parseData = {};
+		mapKeys( data.stats, ( value, key ) => {
```

Comment:
The purpose of `mapKeys` is to reassign the property names of the object, so this can be simplified:

```js
return mapKeys( data.stats, ( value, key ) => camelCase( key ) );
```

https://lodash.com/docs#mapKeys